voltixApp/VoltixApp.xcodeproj/xcshareddata/xcschemes/VoltixApp.xcscheme

```diff
@@ -17,7 +17,7 @@
                BlueprintIdentifier = "DE49A1622B65F6D9000F3AFB"
                BuildableName = "VoltixApp.app"
                BlueprintName = "VoltixApp"
-               ReferencedContainer = "container:VoltixApp.xcodeproj">
```

Comment:
@Rockindash , I am not sure why xcode fighting with this , do you know?

Reply:
I noticed that as well. Sometimes it is low case and sometimes it is capital.

Reply:
So, the closet reference I could find to this issue was this forum
https://forums.developer.apple.com/forums/thread/663975
Seems like, it's how Xcode modifies the scheme to run it over different devices (simulators or physical devices).
We don't need to worry about it since its another of those annoying Xcode's issues. But if needed we can add this file to git ignore.